examples/pytorch/language-modeling/run_clm.py

```diff
@@ -202,18 +203,19 @@ def main():
         datefmt="%m/%d/%Y %H:%M:%S",
         handlers=[logging.StreamHandler(sys.stdout)],
     )
-    logger.setLevel(logging.INFO if training_args.should_log else logging.WARN)
+
+    log_level = training_args.get_process_log_level()
+    logger.setLevel(log_level)
+    datasets.utils.logging.set_verbosity(log_level)
+    transformers.utils.logging.set_verbosity(log_level)
+    transformers.utils.logging.enable_default_handler()
+    transformers.utils.logging.enable_explicit_format()
```

Comment:
interesting. `run_translation.py` didn't have these 2. @sgugger, do we want these in all examples?

Reply:
but they were conditioned by `if training_args.should_log`, so this is a different behavior now.

Reply:
We want them everywhere yes. I don't know if the should_log test really changes anything.

Reply:
@bhadreshpsavani, would you kindly add these 2 lines to `run_translation.py` as well?

and perhaps check if there are others missing these 2?